### PR TITLE
Allow setting `model_name` in the prompt's TOML

### PR DESCRIPTION
**Pull request by cosmastech**
To resolve https://github.com/talkdai/dialog/issues/31

### CODE REVIEW

File: src/llm/__init__.py [hunk 1]
```diff
@@ -18,13 +18,14 @@
 from settings import OPENAI_API_KEY, PROJECT_CONFIG, VERBOSE_LLM
 from sqlalchemy import asc, select
 
+EMBEDDINGS_LLM = OpenAIEmbeddings(openai_api_key=OPENAI_API_KEY)
+PROMPT = PROJECT_CONFIG.get("prompt")
+MODEL_NAME = PROMPT.get("model_name", default="gpt-3.5-turbo")
```
Comment (avelino) [inline src/llm/__init__.py:23]: this configuration should be in the settings package

https://github.com/talkdai/dialog/blob/main/src/settings.py

**Review by avelino — CHANGES_REQUESTED** [review]

File: src/settings.py [hunk 2]
```diff
@@ -11,3 +11,6 @@
     cast=lambda filename: tomllib.loads(Path(filename).read_text()),
     default={},
 )
+
+PROMPT = PROJECT_CONFIG.get("prompt")
+MODEL_NAME = PROMPT.get("model_name", default="gpt-3.5-turbo")
```
Comment (vmesel) [inline src/settings.py:16]:
```
api-web-1  |     MODEL_NAME = PROMPT.get("model_name", default="gpt-3.5-turbo")
api-web-1  |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
api-web-1  | TypeError: dict.get() takes no keyword arguments
```

Can you please fix this?

**Review by vmesel — CHANGES_REQUESTED** [review]

File: src/settings.py (hunk 2 above)
Comment (cosmastech) [reply src/settings.py]: Thanks!

**Review by vmesel — APPROVED** [review]

**Comment by vmesel** [comment]
LGTM! @avelino what are your thoughts?

**Review by avelino — APPROVED** [review]

**Comment by lgabs** [comment]
Later we can maybe put parameters and configs related to the llm configuration into the same `LLM_CONFIG`, which [entered today](https://github.com/talkdai/dialog/pull/59), actually. This will be specially useful when we think about using different llms.

Besides that, it'd be good to indicate about the model_name in the [README](https://github.com/talkdai/dialog#toml-prompt-configuration).

**Comment by vmesel** [comment]
@cosmastech can you resolve the conflicts?

**Comment by cosmastech** [comment]
> @cosmastech can you resolve the conflicts?

Should be all resolved @vmesel

**Comment by avelino** [comment]
thanks @cosmastech

**Pull request merged**